### DISCUSSION TITLE
LIMEが使えない

### DISCUSSION THREAD

**Discussion by takosu2-22** [Chipppppppppp/LIME#228] [Q&A（質問）]
Line（14.16.0） をインストールして、
LSPosed（1.9.3）でLIME （1.10.1） のモジュールを有効化してもLIMEをひらくと
「モジュールが有効化されていません」
と言われてしまい、LIMEを使うことができません。
どうすれば起動できるようになりますか。

**Comment by areteruhiro** [comment]
https://github.com/JingMatrix/LSPosed/releases

**Reply by areteruhiro** [reply]
こちらを使ってください

**Comment by takosu2-22** [comment]
お返事ありがとうございます。
LSPosed（1.10.1）を入れてみたのですが、LIMEの「モジュールは有効化されていません」は出てこないのですが、LINEの設定にLIMEが出できないのですが、
どうすればうまくいきますかね
![Screenshot_20241231-160956](https://github.com/user-attachments/assets/e851aa7e-0395-495d-a030-4f6c1bdb6cdf)
![Screenshot_20241231-160949](https://github.com/user-attachments/assets/7efd0e78-7126-4181-9301-7ea5d5e8427e)

**Comment by areteruhiro** [comment]
LsPosedのアプリを開いて、
ログを削除→lineを強制終了→lineを開く

lsposed→ログ画面をスクショ 
 
スクショ画面をアップロードしてください

**Comment by areteruhiro** [comment]
どこの、14.16.0のline ダウンロードしましたか？
多分それが原因です。

**Comment by takosu2-22** [comment]
お返事ありがとうございます
lineを強制終了して起動してみたのですが、
更新してもログが出ないですね...
（別のモジュールのログは出るのですが...）　

lineはuptodownからインストールしました↓
https://line.jp.uptodown.com/android/download/1029268442
![Screenshot_20241231-165315](https://github.com/user-attachments/assets/a489717e-fb83-41ef-be27-d5e5def40ebb)

**Reply by areteruhiro** [reply]
一度、適用からlineを外し、再起動

また、適用からlineを適用し再起動してみてください。

xapkでも、rootの方はできますので他のapkを利用することも検討してください

**Comment by takosu2-22** [comment]
お返事ありがとうございます
一通り試してみたのですが、（適用＋apk変更）
全部上手くいかなかったので、magiskをアンインストールしてもう一度パッチし直してみます

**Reply by areteruhiro** [reply]
こちらも、漁っていたら5chに以下のような記載がありました。
おそらく、モジュールの焼き直しで、治るかとは思いますが記載しておきます。


本家の最後のLSPosed、今は使えててもGoogle Playシステムアップデートで動かなくなるかも
t.me/LSPosedDiscussion/1674239

Android14 QPR2でのARTの変更が、GooglePlayシステムアップデート経由でAndroid12～Android14 QPR1の端末にも順次配信されているとのこと
(9月4日の情報だけど)
対処法は、A14QPR2やA15Beta4に対応しているJingMatrixやmywalkbなどのLSPosedフォークに乗り換える
もしくは、
adb uninstall com.google.android.art
でARTのアップデートをアンインストールすると本家LSPosedでも動くようになる

**Comment by takosu2-22** [comment]
焼き直しをしてみたら、使えるようになりました。
最後まで対処法を教えていただき誠にありがとうございました

**Reply by areteruhiro** [reply]
多分、magisk 焼き直しまではいかなくても
LsPosed のモジュール無効→新しいモジュール
にしたら行けたかもしれないですね。
